packages/mobile/src/screens/collection-screen/CollectionScreenDetailsTile.tsx

```diff
@@ -225,16 +225,9 @@ export const CollectionScreenDetailsTile = ({
 
   const isReachable = useSelector(getIsReachable)
 
-  const { data: currentUserId } = useCurrentUserId()
-  // Since we're supporting SmartCollections, need to explicitly check that
-  // collectionId is a number before fetching the playlist. -1 is a placeholder,
-  // the request should not go out as the hook is disabled in that case.
-  const { data: collection } = useGetPlaylistById(
-    {
-      playlistId: typeof collectionId === 'number' ? collectionId : -1,
-      currentUserId
-    },
-    { disabled: typeof collectionId !== 'number' }
+  const { data: collection } = useCollection(
+    typeof collectionId === 'number' ? collectionId : -1,
```

Comment:
do we need the -1? could just do `collectionId!` probably

Reply:
no def not, good catch